### PR TITLE
Ts cleanup 2

### LINKED ISSUE #1223: Hide SD card, TS port from Connection TS page

### PR DESCRIPTION
**Pull request by mck1117**
Fixes #1223.  Hides SD card, CAN pins, Tunerstudio port.

Also covers some missing stuff that should've already been hidden (SPI pins).

### CODE REVIEW

**Comment by mck1117** [comment]
...and fixes #1215 too!

File: firmware/config/boards/kinetis/prepend.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+#define ts_show_trigger_comparator true
```
Comment (mck1117) [inline firmware/config/boards/kinetis/prepend.txt]: @rusefi @andreika-git it seems like this doesn't work, is there some trick to get the tool to pick this up?

Comment (andreika-git) [reply firmware/config/boards/kinetis/prepend.txt]: Try putting it inside firmware/config/boards/kinetis/config/rusefi_config_kinetis.txt

Comment (andreika-git) [reply firmware/config/boards/kinetis/prepend.txt]: Or firmware/config/boards/kinetis/config/tunerstudio/kinetis_prefix.txt

Comment (mck1117) [reply firmware/config/boards/kinetis/prepend.txt]: The former suggestion seems to work - thanks!

**Pull request merged**